### PR TITLE
fix: [vmctl] fix support for influx series without tag

### PR DESCRIPTION
**Pull request by jiekun**
### Describe Your Changes

Previously, vmctl expect that tag must exist for each measurement, but it's actually not necessary.

https://github.com/VictoriaMetrics/VictoriaMetrics/blob/f16a58f14c43663ca35093b6583539934bdc6544/app/vmctl/influx/influx.go#L183-L186

This pull request fix it by removing the check. For influx series `measurement1_value1{}`, it will be represented as:
```go
Series{
  Measurement: "measurement1",
  Field:       "value1",
  LabelPairs:  []LabelPair{},
  EmptyTags:   []string{},
}
```
and searched by the following query:
```sql
select "value1" from "measurement1"
``` 
### Checklist

The following checks are **mandatory**:

- [x] My change adheres [VictoriaMetrics contributing guidelines](https://docs.victoriametrics.com/contributing/).

### CODE REVIEW

File: app/vmctl/influx/influx.go [hunk 1]
```diff
@@ -180,11 +180,7 @@ func (c *Client) Explore() ([]*Series, error) {
 			log.Printf("skip measurement %q since it has no fields", s.Measurement)
 			continue
 		}
-		tags, ok := measurementTags[s.Measurement]
-		if !ok {
-			return nil, fmt.Errorf("failed to find tags of measurement %s", s.Measurement)
-		}
-		emptyTags := getEmptyTags(tags, s.LabelPairs)
+		emptyTags := getEmptyTags(measurementTags[s.Measurement], s.LabelPairs)
```
Comment (f41gh7) [inline app/vmctl/influx/influx.go:183]: Looks like it's worth to add fast path check for `getEmptyTags` function:

```
if len(tags) == 0 {
  return nil
}
```

Comment (f41gh7) [reply app/vmctl/influx/influx.go:183]: Also, is it possible that given code panic?

```
func getEmptyTags(tags map[string]struct{}, LabelPairs []LabelPair) []string {
	labelMap := make(map[string]struct{})
	for _, pair := range LabelPairs {
		labelMap[pair.Name] = struct{}{}
	}
         # LabelPairs has some duplicates
         # len(labelMap) = 3
         # len(LabelPairs) = 6
         # result := make([]string, 0, -3)
         result := make([]string, 0, len(labelMap)-len(LabelPairs))
```

Comment (jiekun) [reply app/vmctl/influx/influx.go:183]: Wow thanks for the catch. It's a bug calculating the diff of `tags` and `LabelPairs`. Somehow `len(labelMap)` was used.

File: app/vmctl/influx/influx.go [hunk 2]
```diff
@@ -201,11 +197,16 @@ func (c *Client) Explore() ([]*Series, error) {
...
+		// fast path: the measurement does not contain any tag
+		return nil
+	}
+
 	labelMap := make(map[string]struct{})
 	for _, pair := range LabelPairs {
 		labelMap[pair.Name] = struct{}{}
 	}
-	result := make([]string, 0, len(labelMap)-len(LabelPairs))
+	result := make([]string, 0)
```
Comment (f41gh7) [inline app/vmctl/influx/influx.go:209]:
```suggestion
	var result []string
```

**Review by f41gh7 — APPROVED** [review]
LGTM

**Comment by f41gh7** [comment]
Thanks for contribution!

**Pull request merged**